source/crud/read-operations/retrieve.txt

```diff
@@ -84,8 +93,28 @@ The following shows the output of the preceding query:
 After the owner runs this query, they find two orders that matched the
 criteria.
 
-For a runnable ``find()`` example, see our :doc:`Find Multiple
-Documents </usage-examples/find>` page.
+Find Example: Full File
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The following code is a complete, standalone file that performs a find
+operation to retrieve multiple douments, and a find operation with the first
+method to retrieve a single document. 
+
+.. include:: /includes/crud/example-intro.rst
```

Comment:
S: this kind interrupts the flow since you have it after the code example description; maybe put it in an admonition? Or just at the beginning of this section?

Reply:
Applies to all other inclusions of /includes/crud/example-intro.rst